packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -336,30 +325,29 @@ sample configuration section:
 
 [source,yaml]
 ------------------------------------------------------------------------------
-packetbeat.protocols:
-  http:
+packetbeat.protocols.http:
 
-    # Configure the ports where to listen for HTTP traffic. You can disable
-    # the http protocol by commenting the list of ports.
-    ports: [80, 8080, 8000, 5000, 8002]
+  # Configure the ports where to listen for HTTP traffic. You can disable
+  # the http protocol by commenting the list of ports.
+  ports: [80, 8080, 8000, 5000, 8002]
 
-    # Uncomment the following to hide certain parameters in URL or forms attached
-    # to HTTP requests. The names of the parameters are case insensitive.
-    # The value of the parameters will be replaced with the 'xxxxx' string.
-    # This is generally useful for avoiding storing user passwords or other
-    # sensitive information.
-    hide_keywords: ["pass", "password", "passwd"]
+  # Uncomment the following to hide certain parameters in URL or forms attached
+  # to HTTP requests. The names of the parameters are case insensitive.
+  # The value of the parameters will be replaced with the 'xxxxx' string.
+  # This is generally useful for avoiding storing user passwords or other
+  # sensitive information.
+  hide_keywords: ["pass", "password", "passwd"]
 
-    # Uncomment the following to export a list of extra HTTP headers. By
-    default is none sent.
-    send_headers: ["User-Agent", "Cookie", "Set-Cookie"]
+  # Uncomment the following to export a list of extra HTTP headers. By
+  default is none sent.
+  send_headers: ["User-Agent", "Cookie", "Set-Cookie"]
 
-    # Uncomment the following to export Cookie or Set-Cookie headers. By
-    # default is false.
-    split_coookie: true
+  # Uncomment the following to export Cookie or Set-Cookie headers. By
+  # default is false.
```

Comment:
Elsewhere, I see split_cookie described this way: "If the Cookie or Set-Cookie headers are sent, this option controls whether they are split into individual values."  This seems more accurate and easier to follow than the description given here.